index.html

```diff
@@ -89,6 +89,11 @@ <h1 class="popover__title">
     var directory = [
         {% for entry in site.directory %}
 
+        <!-- id -->
```

Comment:
maybe add a bit more description here (as we are all new to the jekyll stuff)

Reply:
Yeah, you’re right. Also this Liquid templating stuff can be a little tricky sometimes.